tests/test_cases.py

```diff
@@ -0,0 +1,97 @@
+import pandas as pd
+import pytest
+import datatest as dt
+
+import cases
+
+# Set of tests to confirm specific attributes within the raw data being returned
+def test_get_case_data_column_count():
+    # call function
+    actual = cases.get_case_data(start="04-29-2020", end="04-30-2020")
+    actual_col_count = actual.shape[1]
+
+    # set expectations
+    expected_col_count = 12
+
+    assert actual_col_count == expected_col_count
+
+
+# Test for column names and column order
+def test_get_case_data_column_names_and_order():
+    # call function
+    actual = cases.get_case_data(start="05-01-2020", end="05-02-2020")
+    actual_col_names_list = list(actual.columns)
+
+    # set expectations
+    expected_col_names = [
+        "FIPS",
+        "Admin2",
+        "Province_State",
+        "Country_Region",
+        "Last_Update",
+        "Lat",
+        "Long_",
+        "Confirmed",
+        "Deaths",
+        "Recovered",
+        "Active",
+        "Combined_Key",
+    ]
+    # assertion
+    dt.validate(actual.columns, expected_col_names)
```

Comment:
why not use `actual_col_names_list` here?

Reply:
Makes sense. Validating actual_col_names_list and actual.columns does the same thing.